src/main/java/net/datafaker/transformations/JsonTransformer.java

```diff
@@ -125,9 +125,19 @@ private static void value2String(Object value, StringBuilder sb) {
             if (toWrap) {
                 sb.append("\"");
             }
-            for (char c : String.valueOf(value).toCharArray()) {
-                sb.append(ESCAPING_MAP.getOrDefault(c, String.valueOf(c)));
+            int start = 0;
+            final int length = val.length();
+            for (int i = 0; i < length; i++) {
+                final char c = val.charAt(i);
+                final String escapedValue = ESCAPING_MAP.get(c);
+                if (escapedValue == null) {
```

Comment:
I know it means using negative logic but if you check not null then no other condition/branch is needed.

Reply:
makes sense, done